src/blog/2024/11/bridging-modbus-to-mqtt.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Bridging Modbus Data to MQTT using FlowFuse
```

Comment:
Alternatively we could make this UNS focussed, what do you think? `Publishing Modbus data to your UNS with FlowFuse`?

Reply:
let's also consider updating the filename and thus the blog URL path